include/metal/list/sort.hpp

```diff
@@ -46,9 +50,9 @@ namespace metal {
     ///
     /// ### See Also
     /// \see list, reverse, rotate
-    template<typename seq, typename lbd>
+    template<typename seq, typename... lbd>
     using sort = detail::call<
```

Comment:
I considered to customize the code shown by doxygen because we wanted it to read
```
template<typename seq, typename lbd = metal::lambda<metal::less>>>
```
and the only reason not to do so is portability for nonconforming compilers. It would surely improve the clarity of the documentation, but let us first decide if we still want the default ordering at all (see my concern in other comment) before we care about this tweak.

Reply:
Are you sure `template<class seq, class lbd = metal::lambda<metal::less>>>` doesn't work? 
Have you tested? Sometimes MSVC cooperates.

Worst case I agree that we need to customize it for doxygen such that it reads nicely on docs.

Reply:
I had not tested it initially, but in fact there are [failing tests](https://ci.appveyor.com/project/brunocodutra/metal/builds/28065885/job/n97t30q3e0pyaal3#L194) in #107.

Customization of the docs is dangerous though: I can think of a contrived case where our workaround behaves differently. In my mind (and also my search history) there is this error message:
> pack expansion used as argument for non-pack parameter of alias template

What can we do now? I can think of these three options:
 1. try to find another workaround that makes MSVC happy
 1. keep this version and customize the docs (adding a note that the real behavior differs in edge cases)
 1. keep this version, but do not customize the docs

If we choose (3) then the docs are more concise (and contain less potentially confusing notes on contrived edge cases). On the other hand, the clarity of the default argument in code is a strong argument for (2).

What do you think?